src/libstd/env.rs

```diff
@@ -590,6 +590,8 @@ impl ExactSizeIterator for ArgsOs {
 
 /// Returns the page size of the current architecture in bytes.
 #[unstable(feature = "page_size", reason = "naming and/or location may change")]
+#[deprecated(since = "1.3.0",
```

Comment:
This is used by [coroutine-rs](https://github.com/rustcc/coroutine-rs/blob/master/src/stack/stack_protected.rs#L34), would be nice to provide this somewhere since it's os specific and non-trivial to obtain.